include/swift/Basic/PlaygroundOptions.def

```diff
@@ -37,4 +37,7 @@ PLAYGROUND_OPTION(ScopeEvents, "Scope entry/exit events",
 PLAYGROUND_OPTION(FunctionParameters, "Values of function parameters",
   /* enabled by default */ true, /* enabled in high-perf mode */ false)
 
+PLAYGROUND_OPTION(SelectiveTransform, "Only transform explicitly annotated code",
```

Comment:
Nit:  should this say "Only transform explicitly annotated code" since the attribute is currently limited to functions?  Or do we not want to be so specific yet?

Reply:
I kept the description generic on purpose (i.e. not mentioning the annotation directly) to leave the door open for any other methods we may add to control what gets transformed in the future.